mithril.js

```diff
@@ -1706,16 +1706,16 @@
 		var currentTarget = e.currentTarget || e.srcElement
 		var args
 
+		while (currentTarget && !/^a$/i.test(currentTarget.nodeName)) {
```

Comment:
I'm not sure. I would have thought 2 value equality checks would be faster than method invocation or interpolation of a property which may not be there… But at this point I can't find a decent jsPerf to make these kinds of assertions with confidence.